### ISSUE TITLE
413 Request Entity Too Large

### ISSUE THREAD

**Issue by binary64** [phpmyadmin/docker#139]
nginx error `413 Request Entity Too Large`

When trying to upload a 750 MB .sql file with the following config.user.inc.php:-

```php
$cfg['ExecTimeLimit'] = 0; //Now there is no execution time limit

ini_set("upload_max_filesize", "1T");
ini_set("post_max_size", "1T");
ini_set("max_execution_time", "1000");
ini_set("max_input_time", "1000");
ini_set("max_input_vars", 10000);

```

**Comment by nijel** [comment]
The nginx configuration would have to be updated as well:

https://github.com/phpmyadmin/docker/blob/30857caf5c5ef4343e929ef19f062ac065ffae6e/etc/nginx.conf#L41

**Comment by williamdes** [comment]
Implemented in #261 :tada:
